### PR TITLE
Fix callback never call

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

**Comment by nono** [comment]
The build is red

File: src/app/init.coffee [hunk 1]
```diff
@@ -546,13 +546,20 @@ module.exports = class Init
...
 
     setDeviceLocale: ->
         DeviceStatus.initialize()
         unless window.isBrowserDebugging # Patch for browser debugging
             @notificationManager = new Notifications()
-            @serviceManager = new ServiceManager()
+
+            # The ServiceManager is a flag for the background plugin to know if
+            # it's the service or the application, see https://git.io/vVjJO
+            @serviceManager = new ServiceManager() if @app.name isnt 'SERVICE'
```
Comment (nono) [inline src/app/init.coffee]: Maybe `unless @app.name is 'SERVICE'`

File: src/app/replicator/replication_launcher.coffee [hunk 2]
```diff
@@ -41,45 +41,51 @@ module.exports = class ReplicationLauncher
...
-                                log.info 'No dispatcher for ', doc?.docType
-                                next()
-                    , (err) ->
+        if @replication
+            log.debug "Replication is already launched."
+            callback()
+
+        if @config.get('appState') is 'pause'
+            log.debug "Application is paused."
+            callback()
```
Comment (nono) [inline src/app/replicator/replication_launcher.coffee]: No return here? I'm surprised!

**Comment by kosssi** [comment]
@nono if you can merge it to create the new release ;)

**Pull request merged**